docker/README.md

```diff
@@ -0,0 +1,144 @@
+# dGen Docker Usage Guide
+
+By default, the dgen container uses the [Deleware residential dataset](https://oedi-data-lake.s3.amazonaws.com/dgen/de_final_db/dgen_db.sql).
+
+You can customize the dataset, see the [Customizing the Dataset](#customizing-the-dataset) section below.
+
+### Mac and Linux quick start
+
+This quickstart uses docker-compose to run dgen. The default path to store dgen data files and excel configurations is ~/dgen_data/.  This path is shared with your running containers, you can change this path but you will need to edit the `docker-compose.yml` to reflect the data directory of your choice.
+
+Prerequisites assume you are using a Mac and you already installed [Docker Desktop](https://docs.docker.com/desktop/setup/install/mac-install/)
+
+##### Create the data directory
+```bash
+$ mkdir -p ~/dgen_data/
+$ chmod 755 ~/dgen_data/
+$ ls -l ~/dgen_data/ # Its expected to be empty, after starting dgen you will see data files in this location.
+```
+
+##### Startup the dgen containers
+``` bash
+$ cd dgen/docker/
+$ docker-compose up --build -d
+[+] Running 2/2
+ ✔ Container dgen_1     Started 0.1s
+ ✔ Container postgis_1  Started 0.0s
+```
+
+##### Connect to the running containers
+``` bash
+$ docker attach dgen_1 # Attach to dgen environment container
+$ docker attach $(sudo docker ps --filter "name=dgen" --format "{{.ID}}") # If dgen_1 is not found
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ python dgen_model.py # Run scenario
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ exit # to exit
+$ docker-compose up -d # If you exit, you have to re-up the container if you want to re-attach
+```
+
+### Troubleshooting common issues
+
+#### psycopg2.OperationalError: connection to server
+
+Wait 5-10 minutes for the postgres database to finish starting.
+
+#### General errors and issues
+
+Try clearing your `~/dgen_data` and starting over. Make sure to provide time for the datasets to fully download on the re-attempt.
+
+```bash
+$ docker-compose down
+$ rm -f ~/dgen_data/*
+$ docker system prune -a
+$ docker volume prune -f
+```
+
+### Disabling auto-start for the dgen virtual environment
+
+By default, logging into the `dgen` container automatically activates the `dgen` virtual environment.  For ease of use, its recommended to leave this the default.
```

Comment:
```suggestion
By default, logging into the `dgen` container automatically activates the `dg3n` Conda environment.  For ease of use, its recommended to leave this the default.
```
Additionally, would switching "logging into" to "starting" make more sense here?